schema-next.yaml

```
@@ -2,6 +2,13 @@ file_format: 1.1.0
 schema_url: https://opentelemetry.io/schemas/1.31.0
 versions:
   next:
+    all:
+      changes:
+        # https://github.com/open-telemetry/semantic-conventions/pull/1989
+        - rename_attributes:
+            attribute_map:
+              feature_flag.evaluation.reason: feature_flag.result.reason
+              feature_flag.evaluation.variant: feature_flag.result.variant
```

```suggestion
              feature_flag.variant: feature_flag.result.variant
```

Should this be `feature_flag.variant`? There was no `evaluation` for `variant` before.